protostar/starknet/cheatable_execute_entry_point.py

```diff
@@ -149,6 +156,21 @@ def _run(  # type: ignore
                 run_resources=tx_execution_context.run_resources,
                 verify_secure=True,
             )
+            CheatableExecuteEntryPoint.callstack -= 1
+            if CheatableExecuteEntryPoint.callstack == 0:
+                runner.relocate()
+                try:
+                    save_profile(
```

Comment:
I will leave dirty save like this for now, I will improve it later in separate PR